### PR TITLE
Extend Anim Ave dash with issue priority cards

### PR DESCRIPTION
**Pull request by suzyh**
Also adds a library for making it easier to deal with the issue
objects from monorail.

There's a lot of cargo-culting in this patch, so feel free to heavily nitpick in the review!

### CODE REVIEW

File: client/cz-component-dash.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        }
+      },
+
+      attached: function() {
+        registerSource('cz-config', 'components', function(components) {
+          this.set('components', components);
+        }.bind(this));
+        registerSource('cz-config', 'updateSLO', function(updateSLO) {
+          this.set('updateSLO', updateSLO);
+        }.bind(this));
```
Comment (shans) [inline client/cz-component-dash.html:65]: It's probably simpler to put components and updateSLO into the same object in the config, and just issue 1 registerSource on 'cz-config'.

File: client/cz-component-dash.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        box-sizing: border-box;
+      }
+    </style>
+    <template is="dom-repeat" items="{{components}}">
+      <paper-card heading="{{item.cardHeading}}" id='card'>
+        <div class="card-content">
+          <div class='card-flex'>
+            <paper-item>
+              <paper-item-body>
+                <div secondary inner-h-t-m-l="[[issues(item.issues, updateSLO)]]" class='component'></div>
```
Comment (shans) [inline client/cz-component-dash.html:34]: We're technically not supposed to do this :)

Maybe add a TODO to try and convert into templatized HTML at some point?

File: client/cz-issue-priority-dash.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        if (minPriorityA > minPriorityB)
+          return 1;
+        return 0;
+      },
+
+      attached: function() {
+        registerSource('cz-config', 'users', function(users) {
+          this.set('users', users);
+        }.bind(this));
+        registerSource('cz-config', 'updateSLO', function(updateSLO) {
```
Comment (shans) [inline client/cz-issue-priority-dash.html:85]: oh, I see, the updateSLO is shared config. In that case, leave it as it is.

File: client/cz-issue-priority-dash.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        <div class="card-content">
+
+          <div class='card-flex'>
+            <template is="dom-repeat" items="{{users}}"
+                      filter="filterUsers" sort="sortUsers"
+                      observe="issues">
+              <paper-item>
+                <paper-item-body two-line>
+                  <div>{{item.user}}</div>
+                  <div secondary inner-h-t-m-l="[[priorities(item.issues, updateSLO)]]" class='priorities'></div>
```
Comment (shans) [inline client/cz-issue-priority-dash.html:38]: and here too.

**Comment by shans** [comment]
Structure of this LGTM. I haven't checked all the logic but I don't know that I'd be able to offer anything in particular there anyway.

Add the TODOs and I'll merge it!

**Comment by suzyh** [comment]
Done. Added the TODO to the cz-milestones-dash as well.

**Pull request merged**